docs/tutorials/google/advanced_calibration.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<p>@mpharrigan, should the defaults be set to the cycle depths above in Cirq?</p><p> </p><p>Why do you use just one processor?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Not specifying <code>n_processes=1</code> leads to a hanging response on Colab. </p>

Reply:
<p>You can probably see improved performance if you set <code>characterize_{angle}</code>  to false except for zeta, chi, gamma (like is done for floquet). Although floquet is also set to false fo chi</p>

Reply:
<p>Another option is to change the ndefault in XEB wrapper to n_processes=max(1, #of processors - 1)</p>

Reply:
it's currently the multiprocessing default of # of processors